components/Dialogs/examples/DialogsDismissingExampleViewController.m

```diff
@@ -187,17 +191,18 @@ - (void)viewDidLoad {
 
   self.view.backgroundColor = [UIColor whiteColor];
```

Comment:
Is there a reason not to use the color scheme here?

Reply:
I just missed that, it has been fixed.